app_dart/lib/src/model/proto/internal/scheduler.proto

```diff
@@ -57,7 +57,17 @@ message Target {
     // Name of the LUCI recipe to use for the builder.
     optional string recipe = 13;
     // Additional tag for the target.
-    repeated string tags = 14;
+    repeated StringPair tags = 14;
+}
+
+// A string key-value pair. Typically used for tagging.
+//
+// This structure need to be in sync with the StringPair in LUCI infra.
+// See the `go/src/go.chromium.org/luci/buildbucket/proto/common.proto` in the
+// https://source.chromium.org
+message StringPair {
```

Comment:
nit: next id?

Reply:
This structure should not be changed, do we still need to document next id?

Reply:
Good point, SGTM